tests/unit/containers/find-offer/offer-filter-block/offer-filter-list/OfferFilterList.spec.jsx

```diff
@@ -0,0 +1,100 @@
+import { screen, render, fireEvent } from '@testing-library/react'
+
+import OfferFilterList from '~/containers/find-offer/offer-filter-block/offer-filter-list/OfferFilterList'
+
+import { useAppSelector } from '~/hooks/use-redux'
+import { ProficiencyLevelEnum, UserRoleEnum } from '~/types'
+import { selectOption } from '~tests/test-utils'
+
+vi.mock('~/hooks/use-redux', () => ({
+  useAppSelector: vi.fn()
+}))
```

Comment:
Do we really need to mock redux here and why? Why can't we use `renderWithProviders` test utility with `preloadedState`?

Reply:
I refactored this code according to your suggestion. Thank you